cmd/background-heal-ops.go

```diff
@@ -91,7 +91,9 @@ func (h *healRoutine) run(ctx context.Context, objAPI ObjectLayer) {
 			if task.path != slashSeparator && task.path != nopHeal {
 				ObjectPathUpdated(task.path)
 			}
-			task.responseCh <- healResult{result: res, err: err}
+			if task.responseCh != nil {
```

Comment:
responseCh is always `non-nil` allocated once per newHealSequence() and is referenced over into each task as a non-nil value @klauspost there is no reason to add this check AFAICS. 

did you see a crash without it?

Reply:
Some leftover stuff from when I was looking at integrating healing. Removing it for now.